cruise/CruiseInformation.csv

```diff
@@ -360,6 +360,7 @@ AT50-29,Atlantis,2024-08-08T00:00:00,2024-09-09T00:00:00,Cabled (rvdata) Visions
 EK20240814,Elakha,2024-08-14T15:00:00,2024-08-14T23:30:00,Endurance glider 1153-1 recovery and ROV operations
 EK20240815,Elakha,2024-08-15T15:00:00,2024-08-15T18:00:00,Endurance CSPP deployment 
 KH20240821,Kahuna,2024-08-21T10:00:00,2024-08-22T19:30:00,Pioneer 20c glider deployments 
+EK20240830,Elakha,2024-08-30T15:45:00,2024-08-30T19:47:00,Elakha CSPP deployment CE02SHSP-00037
```

Comment:
Minor nitpick: the departure time from the cruise quick look report was 15:52 and here you have it as 15:45.
Not important to change, but calling your attention to it, if you meant for both sources to be the same.

Reply:
changed all places to 15:52 to match quicklook report